app.js

```diff
@@ -12,14 +14,147 @@ function toggleMobileMenu() {
   closeMenu.classList.toggle('show');
 }
 
-// Add an event listener
-toggleMobileMenu.addEventListener('click, mobileMenuList');
-
 // Close function for mobile-menu
 function closeMobileMenu() {
   mobileMenuList.classList.remove('show');
   mobileMenuButton.classList.remove('hidden');
 }
 
-// An event listener
-closeMobileMenu.addEventListener('click', closeMobileMenu);
+mobileMenuButton.addEventListener('click', toggleMobileMenu);
+closeMenu.addEventListener('click', closeMobileMenu);
+
+const portfolioItemsDetail = [
+  {
+    id: '0',
+    name: 'Tonic',
+    year: '2015',
+    text: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry.Lorem Ipsum has, a galley of type and scrambled it 1960s. Lorem Ipsum is simplydummy text of the printing and typesetting industry.',
+    title: 'CANOPY',
+    stack: 'Back End Dev',
+    workStack: ['HTML', 'CSS', 'JavaScript'],
+    images: 'images/natureDesktop.png',
+    source: '#',
+    live: '#',
+  },
+
+  {
+    id: '1',
+    name: 'Multi-Post Stories',
+    year: '2015',
+    text: 'Lorem Ipsum is simply dummy text of the printing and  Lorem Ipsum is simplydummy text of the printing and typesetting industry.',
+    title: 'FACEBOOK',
+    stack: 'Full Stack Dev',
+    workStack: ['HTML', 'Ruby on Rails', 'CSS', 'Javascript'],
+    images: 'images/storiesDesktop.png',
+    source: '#',
+    live: '#',
+  },
+
+  {
+    id: '2',
+    name: 'Facebook 360',
+    year: '2015',
+    text: 'Lorem Ipsum is simply dummy text of and scrambled it 1960s. Lorem Ipsum is simplydummy text of the printing and typesetting industry.',
+    title: 'Facebook',
+    stack: 'Full Stack Dev',
+    workStack: ['HTML', 'Ruby on Rails', 'CSS', 'Javascript'],
+    images: 'images/Facebook360Desktop.png',
+    source: '#',
+    live: '#',
+  },
+
+  {
+    id: '3',
+    name: 'Uber Navigation',
+    year: '2015',
+    text: 'Lorem Ipsum is simply dummy text of a galley of type and scrambled it 1960s. Lorem Ipsum is simplydummy text of the printing and typesetting industry.',
+    title: 'Uber',
+    stack: 'Lead Developer',
+    workStack: ['HTML', 'Ruby on Rails', 'CSS', 'Javascript'],
+    images: 'images/UberDesktop.png',
+    source: '#',
+    live: '#',
```

Comment:
- The project requirement states that your popup should be displayed dynamically using this highlighted project array. Currently your popup isn't dynamically rendered as it should be and I kindly suggest you make this addition to your project. 👍 
- * ## Project requirement
![dy](https://github.com/Drew246/Portfolio-setup-and-mobile-first/assets/89055502/a8a4ee63-19ce-41e4-8d39-ba61f02abf9f)